providers/resend/src/lib/resend.provider.ts

```diff
@@ -26,7 +27,9 @@ export class ResendEmailProvider implements IEmailProvider {
     options: IEmailOptions
   ): Promise<ISendMessageSuccessResponse> {
     const response: any = await this.resendClient.sendEmail({
-      from: options.from || this.config.from,
+      from: this.config?.senderName
+        ? `${this.config?.senderName} <${options.from || this.config.from}>`
+        : options.from || this.config.from,
```

Comment:
Do you think for better visibility maybe we can extract some of it to variables? For example:


const fromAddress = options.from || this.config.from;

....
from: this.config?.senderName
        ? `${this.config?.senderName} <${fromAddress}>`
        : fromAddress,
....

Reply:
Sure @scopsy Please check commit https://github.com/novuhq/novu/pull/4237/commits/da058e81b25bc944c8ac3a1ae8f783d1678b1b41